R-package/src/xgboost_R.cc

```diff
@@ -20,7 +20,6 @@
 #include "../../src/common/threading_utils.h"
 
 #include "./xgboost_R.h"  // Must follow other includes.
-#include "Rinternals.h"
```

Comment:
This one was also possibly-problematic because some compilers will treat it as "search for a file named `Rinternals.h` in the project directory".

e.g., from the `gcc` docs (https://gcc.gnu.org/onlinedocs/cpp/Include-Syntax.html):

> `#include <file>`
> *This variant is used for system header files. It searches for a file named file in a standard list of system directories. You can prepend directories to this list with the `-I` option (see [Invocation](https://gcc.gnu.org/onlinedocs/cpp/Invocation.html)).*

> `#include "file"`
> *This variant is used for header files of your own program. It searches for a file named file first in the directory containing the current file, then in the quote directories and then the same directories used for `<file>`. You can prepend directories to the list of quote directories with the -iquote option.*